hard-code reminders with li classes

stephanieea opened the pull request:
## Purpose

The problem is that we need to display reminders on the screen.  
 Closes [Issue #1SeeAllReminders](https://github.com/turingschool-projects/1610-remember-9/issues/1])

Closes Issue#1

## Approach

We added an array of reminders and added the classname to the li tag.  

### Learning

We referenced the ember-groceries lesson from school and the ember docs to ensure we were making changes in the correct places.  

### Open Questions and Pre-Merge TODOs

We hard coded the reminders but weren't sure if your intention was to create a ui so that the user could enter reminders.  

### Test coverage 

We made the first acceptance reminder list test, that was already written, pass.

stephanieea committed:
hard-code reminders with li classes

martensonbj commented on tests/unit/routes/index-test.js:
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```
This is file is not necessary - we can delete it.

martensonbj commented on app/routes/index.js:
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
```
This logic should live in a `reminders` route. Generate this with `ember g route reminders` and rethink how we are handing our model to the template. The index route/template should only be used if we need default material on a page, in this issue we want our logic to be associated directly with a `reminders` route. 

Secondly, Mirage is generating 5 reminders for you and popping them into your data store. Once you have the reminders route up created, look into how to find records from the store and pass them to the `reminders.hbs` template.

martensonbj replied:
AKA you shouldn't have to hard code in any reminders at this point.

martensonbj reviewed (changes requested):
Few changes - let me know if you have questions on my comments.

Tman22 commented on app/templates/index.hbs:
```diff
@@ -0,0 +1,5 @@
+<h1>REMINDERS</h1>
+
+{{#each model as | reminder |}}
+  <ul><li class='spec-reminder-item'>{{reminder}}</li></ul>
```
The <ul> tag should be outside the {{each}}. Currently you are making 5 <ul> with a single <li> in each.

martensonbj replied:
As a follow up - this is ok if you want to treat each reminder as a UL< although I think it makes more sense to treat each reminder as an article with a title (say maybe h2), date (p) and notes (p).

stephanieea committed:
add structure for singular reminder item, model for reminder, and, route for reminders.

stephanieea committed:
add passing integration test for reminder item component

stephanieea commented:
@Tman22 @martensonbj We moved the reminder logic to the reminder route.  We also moved the ul tag outside of the .spec-reminder-item class.  We deleted the unnecessary file.  We added the changes to pull in reminders from mirage. 🎉

stephanieea committed:
fix linting

stephanieea committed:
rename model in models reminder-test

stephanieea committed:
remove html from hbs file per last comment

martensonbj commented on tests/integration/components/reminder-item-test.js:
```diff
@@ -0,0 +1,18 @@
+import { moduleForComponent, test } from 'ember-qunit';
```
👍

martensonbj commented on app/router.js:
```diff
@@ -7,6 +7,10 @@ const Router = Ember.Router.extend({
 Router.map(function() {
+//  this.route('reminders');
+  this.route('reminders', { path: '/' });
 });
```
Good adjustment! Let's clean up the extra lines and comments here now.

martensonbj commented:
Looks good! Keep an eye on extra lines/spacing in the next PRs
